decidim-proposals/lib/decidim/proposals/seeds.rb

```diff
@@ -11,13 +12,37 @@ def initialize(participatory_space:)
         @participatory_space = participatory_space
       end
 
-      # rubocop:disable Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
       def call
-        admin_user = Decidim::User.find_by(
-          organization: participatory_space.organization,
-          email: "admin@example.org"
-        )
+        component = create_component!
+
+        5.times do |n|
+          proposal = create_proposal!(component:)
+
+          emendation = create_emendation!(proposal:) if proposal.state.nil?
+
+          (n % 3).times do |_m|
+            create_proposal_votes!(proposal:, emendation:)
+          end
+
+          (n % 3).times do
+            create_proposal_notes!(proposal:)
+          end
+
+          Decidim::Comments::Seed.comments_for(proposal)
 
+          create_collaborative_draft!(component:)
+        end
+
+        update_traceability!(component:)
+      end
+
+      def organization
+        @organization ||= participatory_space.organization
+      end
+
+      def admin_user = Decidim::User.find_by(organization:, email: "admin@example.org")
```

Comment:
Consistency ( see organization method. Also used in 2 places ( including a loop ) . 
```suggestion
      def admin_user
        @admin_user ||= Decidim::User.find_by(organization:, email: "admin@example.org")
      end 
```